### DISCUSSION TITLE
Welcome to SadConsole Discussions!

### DISCUSSION THREAD

**Discussion by Thraka** [Thraka/SadConsole#263] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * The SadConsole community is also on discord. Use this invite: https://discord.gg/pAFNKYjczM

**Comment by melance** [comment]
The discord invite appears to be invalid.

![image](https://user-images.githubusercontent.com/8439866/122051159-e203d080-cda9-11eb-9b99-a2755e510833.png)

**Reply by Thraka** [reply]
Thanks, I updated it.

**Reply by nsxdavid** [reply]
It still does that.

**Reply by Thraka** [reply]
@nsxdavid I just tried it and it opens discord for me. What is your user name, I'll just add you directly
